### PR TITLE
Use `sphinx.ext.intersphinx` to link to other projects' Sphinx documentation

### PR DESCRIPTION
**Pull request by edgarrmondragon**
### Description

Follow-on https://github.com/sqlalchemy/sqlalchemy/pull/11762.

### Checklist

This pull request is:

- [x] A documentation / typographical / small typing error fix
	- Good to go, no issue or tests are needed
- [ ] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

**Comment by CaselIT** [comment]
Hi,

I'm not sure what's the advantage here? Now reading the source file makes it very hard to understand where the link point to

**Comment by zzzeek** [comment]
hi -

I've had lots of problems with intersphinx in the past and ive had to stop using it in places we used to use it.   in the past it's been largely if not completely broken.

in this case these are top-level links to other docs for other projects.  I see no reason to use intersphinx for these, it just opens us up to more random problems that happen silently in our doc builds without us knowing.

**Comment by edgarrmondragon** [comment]
Gotcha, just wanted to open as a suggestion to prevent things like https://github.com/sqlalchemy/sqlalchemy/pull/11762 in the future. But you're right, this is just links to project home pages so feel free to close 🙂.

File: lib/sqlalchemy/dialects/oracle/cx_oracle.py [hunk 1]
```diff
@@ -61,8 +61,7 @@
     engine = create_engine("oracle+cx_oracle://scott:tiger@myalias/?encoding=UTF-8&nencoding=UTF-8")
 
 Users of Oracle Cloud should use this syntax and also configure the cloud
-wallet as shown in cx_Oracle documentation `Connecting to Autononmous Databases
-<https://cx-oracle.readthedocs.io/en/latest/user_guide/connection_handling.html#autonomousdb>`_.
+wallet as shown in cx_Oracle documentation :external+cx-oracle:ref:`autonomousdb`.
```
Comment (zzzeek) [inline lib/sqlalchemy/dialects/oracle/cx_oracle.py:64]: in your PR at #11762 , intersphinx doesnt help much unless we are trying to guard against the page itself changing.   the link that's there was probably taken from the URL bar and we simply didnt know the link had a dedicated anchor.

**Comment by CaselIT** [comment]
Closing for now.. thanks in any case!

**Pull request closed without merging**